src/reference/asciidoc/kafka.adoc

```diff
@@ -2660,6 +2660,46 @@ You can trust other (or all) packages by adding trusted packages with the `addTr
 If you receive messages from untrusted sources, you may wish to add only those packages you trust.
 To trust all packages, you can use `mapper.addTrustedPackages("*")`.
 
+Starting with version 2.2.5, you can specify that certain string-valued headers should not be mapped using JSON, but to/from a raw `byte[]`.
+This is useful when communicating with systems that are not aware of the mapper's JSON format.
+The `AbstractKafkaHeaderMapper` has new properties `mapAllStringsOut` when set to true, all string-valued headers will be used to
+`byte[]` using the `charset` property (default `UTF-8`).
```

Comment:
Combine with the previous line, please.

Reply:
Sorry, I am not sure what you mean (although I do see a correction I need to make).

Reply:
I mean this: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
Doh - got it.